src/bss.rs

```diff
@@ -80,30 +72,29 @@ impl ParseNlAttr for Bss {
     fn parse(&mut self, handle: AttrHandle<Nl80211Attr>) -> Bss {
```

Comment:
This is sort of a weird way to define this, versus say, an associated function or a `TryFrom` / `From` impl. As is it doesn't appear that this would be a huge lift to refactor. What do you think?

Reply:
Agree, implemented an associated function